enhavo/dashboard-bundle/0.8/assets/enhavo/dashboard.ts

```diff
@@ -0,0 +1,2 @@
+import Application from "@enhavo/dashboard/Dashboard/DashboardApplication";
+Application.getVueLoader().load(() => import("@enhavo/dashboard/Dashboard/Components/ApplicationComponent.vue"));
```

Comment:
Why is this file needed in the recipe?

Reply:
This is the entry point for the dashboard. It will be used to add js and css or load further assets from other enhavo bundles

Reply:
But you cannot be sure this is added to webpack.. It make little sense. 

Instead of asking the user to include this file in webpack.js, then you may ask them to include the file from your bundle instead.